### PR TITLE
setting role to restart service when its run/updated

### PR DESCRIPTION
**Pull request by joshuafernandes**
- setting role to restart service when its run/updated
- made the role idempotent

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | `pantheon_user` | pantheon | Pantheon user |
 | `pantheon_group` | pantheon | Pantheon group |
 | `pantheon_version` | 1.1.4 | Current stable version of Pantheon |
+| `pantheon_download_url` | https://bintray.com/consensys/pegasys-repo/download_file?file_path=pantheon-{{ pantheon_version }}.tar.gz | The download tar.gz file used. You can use this if you build pantheon from source and publish to an internal repo. |
```
Comment (ajsutton) [inline README.md]:
```suggestion
| `pantheon_download_url` | https://bintray.com/consensys/pegasys-repo/download_file?file_path=pantheon-{{ pantheon_version }}.tar.gz | The download tar.gz file used. You can use this if you need to retrieve pantheon from a custom location such as an internal repository. |
```

File: tasks/install.yml [hunk 2]
```diff
@@ -71,9 +79,3 @@
     state: link
   become: true
 
-- name: Add pantheon to system-wide $PATH
-  copy:
-    dest: "{{ pantheon_profile_file }}"
-    content: 'PATH=$PATH:{{ pantheon_current_dir }}/bin'
-  become: true
-  when: ansible_os_family != "Darwin"
```
Comment (ajsutton) [inline tasks/install.yml:79]: Did we mean to lose this and not have pantheon on the path?

**Review by ajsutton — APPROVED** [review]
Actually reads quite cleanly and LGTM.

File: tasks/install.yml (hunk 2 above)
Comment (joshuafernandes) [reply tasks/install.yml:79]: Yup, its unused so figured I'll clean it up

**Pull request merged**